tools/build-zephyr.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# Copyright © 2016 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# clone zephyr repo
+git clone https://gerrit.zephyrproject.org/r/zephyr
+cd zephyr
+
+# download the zephyr_sdk installer script
+wget https://nexus.zephyrproject.org/content/repositories/releases/org/\
```

Comment:
that wouldn't work.  I tried looking into ways of splitting that as well as indenting, but couldn't find a nice looking one in bash...i am splitting it but i cannot indent it as well (afaik)